blog/_posts/2017-09-04-gsoc-ode.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title:  "GSoC 2017: Native Julia ODE solvers"
```

Comment:
Far too generic of a title. The title should clearly indicate that this is about second order ODEs and BVPs